### ISSUE TITLE
docker compose file is not available

### ISSUE THREAD

**Issue by peratik** [openblocks-dev/openblocks#392]
Hi

I have been trying to download this file for a few days:
`https://cdn-files.openblocks.dev/docker-compose.yml`

It is not available:

`curl: (6) Could not resolve host: cdn-files.openblocks.dev`

![image](https://github.com/openblocks-dev/openblocks/assets/6568353/a76081b5-03f4-4c92-802d-3dc84fda9f27)

Please advise.

Thank you

**Comment by Rikezenho** [comment]
Same here.

**Comment by travisaustin** [comment]
I found it here: https://github.com/openblocks-dev/openblocks/tree/develop/deploy/docker

**Comment by VinnyChase28** [comment]
any updates on this? 

MacBook-Pro:~/Projects/openblocks$ docker-compose up -d
[+] Building 0.0s (0/0)                                                                                                                                      
[+] Running 0/0
 ⠋ Container openblocks  Starting                                                                                                                       0.0s 
Error response from daemon: driver failed programming external connectivity on endpoint openblocks (a8cb0d056a5e2570039b3588a0bbcdbaee816f9a62f74c57e42fdf922bfa3945): Bind for 0.0.0.0:3000 failed: port is already allocated

**Comment by henriquekuwai** [comment]
https://github.com/lowcoder-org/lowcoder is a working fork and apparently is still maintained.

**Comment by FalkWolsky** [comment]
Yes. We loved Openblocks so much, that we put our heads together and continued the project.
Next to the Opensource Versions everyone can use also the free cloud offering at https://app.lowcoder.cloud
